images/scripts/debian.setup

```diff
@@ -99,6 +99,11 @@ wireguard-tools \
 xfsprogs \
 "
 
+# These are the packages we don't want.
+DENIED_PACKAGES="\
+selinux-policy-default-
```

Comment:
Could we limit this to debian-testing? It doesn't get in the way on the other OSes, so we shouldn't modify more than necessary.

Reply:
Sure, but it might get in the way in the future... less selinux is more, no? :-)

Reply:
OK, not a strong opinion. I even wonder what brings it in.